### DISCUSSION TITLE
Question about input and output flow in Technosphere flows and Biosphere flows.

### DISCUSSION THREAD

**Discussion by OliverLyon** [LCA-ActivityBrowser/activity-browser#1402] [Q&A]
In terms of the input and output flow in the LCI I constructed. How do I transform that to Technosphere flows and Biosphere flows. Do I keep input as positive values and output as negative values or is there any other way to do it? Can anyone briefly explain this. Thanks a lot.

**Comment by OliverLyon** [comment]
I also have a question about multiple products and by products in one process. Is there any way I can add an own defined product in the activity?

**Comment by marc-vdm** [comment]
Hi @OliverLyon I am not sure what exactly you are asking. But you can find the Technosphere and Biosphere flows of an LCA calculation in the `LCA results` tab. Perhaps it would help you to follow the [tutorial](https://github.com/LCA-ActivityBrowser/activity-browser/wiki/Tutorials#your-first-lca) first?
